sklearn/utils/tests/test_validation.py

```diff
@@ -1213,3 +1213,85 @@ def test_check_sparse_pandas_sp_format(sp_format):
     assert sp.issparse(result)
     assert result.format == sp_format
     assert_allclose_dense_sparse(sp_mat, result)
+
+
+def make_types_table():
```

Comment:
I would prefer not to have this function at all, we do not need to check every single pandas dtype combinations.

Reply:
ok